RN2xxx.device.nut

```diff
@@ -37,7 +37,7 @@ class RN2xxx {
     static FIRST_ASCII_PRINTABLE_CHAR = 32;
     static RN2903_BANNER = "RN2903";
     static RN2483_BANNER = "RN2483";
-    static INIT_TIMEOUT = 5;
+    static RESET_TIMEOUT = 5;
```

Comment:
I find it a good practice to use units names in the suffix, ie. _SEC, _MS. Helps to maintain and review the code.